sorting/recursive_bubble_sort.cpp

```diff
@@ -0,0 +1,165 @@
+/**
+ * @file recursive_bubble_sort.cpp
+ * @author Aditya Prakash (https://adityaprakash.tech)
+ * @brief This is an implementation of recursive version of Bubble sort
```

Comment:
Provide a Wikipedia link in Markdown format (if available/possible).
If there's no Wikipedia link, add another web source for algorithm explanation. 🙂